ibis/backends/impala/tests/test_exprs.py

```diff
@@ -508,41 +508,42 @@ def test_aggregations(alltypes):
     agged_table.execute()
 
 
-def test_analytic_functions(alltypes):
-    t = alltypes.limit(1000)
-
-    g = t.group_by('string_col').order_by('double_col')
-    f = t.float_col
-
-    exprs = [
-        f.lag(),
-        f.lead(),
-        f.rank(),
-        f.dense_rank(),
-        f.percent_rank(),
-        f.ntile(buckets=7),
-        f.first(),
-        f.last(),
-        f.first().over(ibis.window(preceding=10)),
-        f.first().over(ibis.window(following=10)),
-        ibis.row_number(),
-        f.cumsum(),
-        f.cummean(),
-        f.cummin(),
-        f.cummax(),
-        # boolean cumulative reductions
-        (f == 0).cumany(),
-        (f == 0).cumall(),
-        f.sum(),
-        f.mean(),
-        f.min(),
-        f.max(),
-    ]
-
-    proj_exprs = [expr.name('e%d' % i) for i, expr in enumerate(exprs)]
-
-    proj_table = g.mutate(proj_exprs)
-    proj_table.execute()
+# FIXME(kszucs)
```

Comment:
Re-enable this test case. Also have a backendless version among the expression tests.

Reply:
Done.